recipe/meta.yaml

```
@@ -1,50 +1,57 @@
+{% set name = "importlib_resources" %}
 {% set version = "6.1.1" %}
 
 package:
-  name: importlib_resources
+  name: {{ name }}-suite
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/i/importlib_resources/importlib_resources-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 3893a00122eafde6894c59914446a512f728a0c1a45f9bb9b63721b6bacf0b4a
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-
-requirements:
-  host:
-    - python >=3.8
-    - pip
-    - setuptools_scm >=3.4.1
-    # setuptools_scm[toml]
-    - toml
-  run:
-    - python >=3.8
-    - zipp >=3.1.0
-  run_constrained:
-    - {{ pin_subpackage("importlib-resources", max_pin="x.x.x") }}
-
-test:
-  requires:
-    - pip
-  commands:
-    - pip check
-  imports:
-    - importlib_resources
+  skip: true  # [py<38]
 
 outputs:
   - name: importlib_resources
-  - name: importlib-resources
-    build:
-      noarch: python
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
     requirements:
       host:
-        - python >=3.8
+        - python
+        - pip
+        - setuptools
+        - setuptools_scm >=3.4.1
+        # setuptools_scm[toml]
+        - toml
+        - wheel
       run:
-        - {{ pin_subpackage("importlib_resources", max_pin="x.x.x") }}
-        - python >=3.8
+        - python
+        - zipp >=3.1.0  # [py<310]
+    test:
+      requires:
+        - pip
+      commands:
+        - pip check
+      imports:
+        - importlib_resources
+
+  - name: importlib-resources
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
```

```suggestion
    script: install.bat  # [win]
    build:
      noarch: python
```
importlib-resources still should be a noarch python package

---

Why so?

---

Also, since you use `{{ pin_subpackage("importlib_resources", max_pin="x.x.x", exact=True) }}` in the run section, you don't need an installation script.

---

done